src/app/rosetta/config.json

```diff
@@ -8,5 +8,262 @@
+        "sk": null
+      },
+      {
+        "balance": "11550000.000000000",
+        "delegate": "B62qqN8cg6GNUVMcSGfyirfqkB5tcyb5h3PeWtbNkNZiBRqK8SmqkxZ",
+        "pk": "B62qjXZAwRw3USLSP9k6A6EKYHKgXbWHqF4KAZrJPhaiH526Hr1QdHy",
+        "sk": null
+      },
+      {
+        "balance": "0.000000000",
+        "delegate": null,
+        "pk": "B62qqN8cg6GNUVMcSGfyirfqkB5tcyb5h3PeWtbNkNZiBRqK8SmqkxZ",
+        "sk": null
+      },
+      {
+        "balance": "11550000.000000000",
+        "delegate": "B62qqVpQv5Uwih1hnn8AyfAVJHcJkufp5KYp2hEnXNSzAoJxXdGuqWn",
+        "pk": "B62qpjYmxjbCF1fa1ZcKG7qitLrGKfRg449vtwYMddR5GcXwjv2y4iW",
+        "sk": null
+      },
+      {
+        "balance": "0.000000000",
+        "delegate": null,
+        "pk": "B62qqVpQv5Uwih1hnn8AyfAVJHcJkufp5KYp2hEnXNSzAoJxXdGuqWn",
+        "sk": null
+      },
+      {
+        "balance": "11550000.000000000",
+        "delegate": "B62qpKetXb5azsSYyRPTECzQ7XnUAs5ZQdKU35hBrLY5moRZ5He7KsZ",
+        "pk": "B62qnvKzBMR6rNb4KZpQ7PTvtmGM22iXe9B4MqFUuwPzuVKbzTk74Nb",
+        "sk": null
+      },
+      {
+        "balance": "0.000000000",
+        "delegate": null,
+        "pk": "B62qpKetXb5azsSYyRPTECzQ7XnUAs5ZQdKU35hBrLY5moRZ5He7KsZ",
+        "sk": null
+      },
+      {
+        "balance": "11550000.000000000",
+        "delegate": "B62qixMZ7gydLMKjRVNwuNHtjCkBkZg4ZhWMn8o9PxAbNvjFraabVau",
+        "pk": "B62qm55VdHAJCR6V73iXjEQwCvPECd6ANKUjfewpFs5dRYgNRuBSVir",
+        "sk": null
+      },
+      {
+        "balance": "0.000000000",
+        "delegate": null,
+        "pk": "B62qixMZ7gydLMKjRVNwuNHtjCkBkZg4ZhWMn8o9PxAbNvjFraabVau",
+        "sk": null
+      },
+      {
+        "balance": "11550000.000000000",
+        "delegate": "B62qqJC6R9KAmeabThLYQmAqUxDccK341jRgo9VZN1UFsqLvpNFdzjP",
+        "pk": "B62qkjhMRSEBargcS5QutGJrxwboACxhaRKGZ3DWi7FdRg8kzQhkAzz",
+        "sk": null
+      },
+      {
+        "balance": "0.000000000",
+        "delegate": null,
+        "pk": "B62qqJC6R9KAmeabThLYQmAqUxDccK341jRgo9VZN1UFsqLvpNFdzjP",
+        "sk": null
+      },
+      {
+        "balance": "11550000.000000000",
+        "delegate": "B62qqdFYzzK7T3gRsAwZmUoMVwJJHMZ9GFCuLpZkrARqSsuMyKqji6J",
+        "pk": "B62qpr73MUU2k4zEgbp2zW2PhXvpXRnEEkRmTZZUAz2VX8PHp4Dg4FK",
+        "sk": null
+      },
+      {
+        "balance": "0.000000000",
+        "delegate": null,
+        "pk": "B62qqdFYzzK7T3gRsAwZmUoMVwJJHMZ9GFCuLpZkrARqSsuMyKqji6J",
+        "sk": null
+      },
+      {
+        "balance": "11550000.000000000",
+        "delegate": "B62qrzGZGFwth7HyN7Xb3yNsqzzTDyQp6zruVgSirE2JUqW2sqiYYLr",
+        "pk": "B62qmGKApiVniXQP1jZx1Fa3zBj4t2h1gERd8j79gdm5X7TjM6KzF8f",
+        "sk": null
+      },
+      {
+        "balance": "0.000000000",
+        "delegate": null,
+        "pk": "B62qrzGZGFwth7HyN7Xb3yNsqzzTDyQp6zruVgSirE2JUqW2sqiYYLr",
+        "sk": null
+      }
+    ],
+    "name": "release",
+    "num_accounts": 250
+  }
```

Comment:
we'll need to make a new config.json for mainnet too -- I think we should probably build some script that can digest the genesis ledger file, that way we can easily migrate it during a hardfork (cc @deepthiskumar to keep note of when we do a hardfork)

Reply:
doesn't have to be this PR that we make that config, just flagging as a TODO